contracts/beacon/Voucher.sol

```diff
@@ -21,7 +22,7 @@ import {IVoucher} from "./IVoucher.sol";
  * @title Implementation of the voucher contract.
  * Deployed along the Beacon contract using "Upgrades" plugin of OZ.
  */
-contract Voucher is OwnableUpgradeable, IVoucher {
+contract Voucher is Initializable, IVoucher {
```

Comment:
Do you update that to get a custom error message with `onlyOwner`?

Reply:
I update this to remove the ability to transfer Ownership

Reply:
Cool thanks!